package.json

```diff
@@ -9,8 +9,7 @@
     ]
   },
   "scripts": {
-    "next:start": "yarn workspace @fuel-starter-kit/nextjs dev",
-    "next:contract:typings": "yarn workspace @fuel-starter-kit/nextjs contract:typings",
+    "next:start": "yarn workspace @fuel-starter-kit/nextjs contract:typings & yarn workspace @fuel-starter-kit/nextjs dev",
```

Comment:
Minor thing, but I think we could use `&&` instead of `&`, so it runs in sequence.

![image](https://user-images.githubusercontent.com/2486142/219408758-c26b1907-4ca6-4c56-a7df-4218b6b2f226.png)

Reply:
updated, thank you so much this is very helpful!